spec/prometheus/client/data_stores/direct_file_store_spec.rb

```diff
@@ -32,6 +32,31 @@
                          metric_type: :counter,
                          metric_settings: { some_setting: true })
     end.to raise_error(Prometheus::Client::DataStores::DirectFileStore::InvalidStoreSettingsError)
+
+    # :most_recent aggregation can only be used for gauges
```

Comment:
I'd prefer this to be done with RSpec methods describing the tests rather than a single comment at the top of a bunch of `expect`s.

Reply:
i'm not really sure what shape the code you're thinking of has, sorry.
to me this is doing that, if we remove the comment, this is pretty much how the rest of the test case is written.

Can you mock a quick example of what you mean by "done with RSpec methods"?